driver-core/src/main/com/mongodb/internal/connection/DefaultServerMonitor.java

```diff
@@ -75,12 +87,17 @@
         monitor = new ServerMonitorRunnable();
         monitorThread = new Thread(monitor, "cluster-" + this.serverId.getClusterId() + "-" + this.serverId.getAddress());
         monitorThread.setDaemon(true);
+        roundTripTimeMonitor = new RoundTripTimeRunnable();
+        roundTripTimeMonitorThread = new Thread(roundTripTimeMonitor,
```

Comment:
Maybe something for the future - but having an easy way to remove the rtt logs from the main logs when debugging would be good / something to document.

Reply:
`RoundTripTimeRunnable` doesn't currently log.  It does get a unique thread name though.

Reply:
ACK